gcc/testsuite/rust/compile/torture/unit_type3.rs

```diff
@@ -0,0 +1,6 @@
+struct S;
+
+fn main() {
+    let s = S;
+    // { dg-warning "unused name" "" { target *-*-* } .-1 }
```

Comment:
nit: If you don't care about the warning you can also use `let _s = S;` since that's working now

Reply:
good point will fix.